shared/common-adapters/header-hoc/index.desktop.tsx

```diff
@@ -78,6 +78,10 @@ function HeaderHoc<P extends {}>(WrappedComponent: React.ComponentType<P>) {
   return HH
 }
 
+export const HeaderHocWrapper = (props: Props & {children: React.ReactNode}) => {
```

Comment:
I don't understand how this does anything.

Reply:
it doesn't, just like the hoc